### DISCUSSION TITLE
Can rest.nvim integrates with markdown file?

### DISCUSSION THREAD

**Discussion by Catizard** [rest-nvim/rest.nvim#374] [Q&A]
I'm currently writing a api doc with markdown.And I think it would be great if I can write something like :
```markdown
## name blabla

example:
post http://localhost/...

{
}

```

**Comment by boltlessengineer** [comment]
Not at this moment. But would be possible using tree-sitter’s language injection

**Comment by gorillamoe** [comment]
At kulala.nvim we support markdown, because I also use this quite heavily for docs.

You can do something like this in your markdown file


``\`http
post http://localhost/...

{
}
``\`

When you then are in the http code block you can run it normally via `lua require("kulala").run()`.

Anyway, would be great if rest.nvim would also support this.

**Reply by Catizard** [reply]
Thanks for the recommendation, I would try it the next time writing api doc 👍
